airflow/config_templates/config.yml

```diff
@@ -2130,6 +2130,14 @@ scheduler:
       type: boolean
       example: ~
       default: "False"
+    scheduler_health_check_server_host:
+      description: |
+        When you start a scheduler, airflow starts a tiny web server
+        subprocess to serve a health check on this host
+      version_added: 2.8.0
+      type: string
+      example: ~
+      default: "0.0.0.0"
```

Comment:
Should the default **continue** to be 0.0.0.0 ? For me it looks like we might choose no default and use IP reverse-dnsed from hostname callable: https://airflow.apache.org/docs/apache-airflow/stable/configurations-ref.html#hostname-callable maybe?). That would be quite a bit more secure default and only slightly risky when it comes to backwards compatibility.

Reply:
A bit. But we do allow breaking changes "because security" ....

Reply:
(just saying :D) . Not very strong on it